build.gradle.kts

```diff
@@ -341,12 +346,73 @@ tasks {
     return buildCodyDir
   }
 
+  fun copyProtocol(): Unit {
+    // This is just a temporary solution to make it easier to use certain generated protocol
+    // messages.
+    // Ultimately this should entirely be replaced by use of the generated protocol.
+    if (!isForceProtocolCopy) {
+      return
+    }
+    val codyDir = downloadCody()
+    val sourceDir =
+        codyDir.resolve(
+            Paths.get(
+                    "agent",
+                    "bindings",
+                    "kotlin",
+                    "lib",
+                    "src",
+                    "main",
+                    "kotlin",
+                    "com",
+                    "sourcegraph",
+                    "cody",
+                    "protocol_generated")
+                .toString())
+    val targetDir =
+        layout.projectDirectory.asFile.resolve(
+            "src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated")
+
+    targetDir.deleteDirectoryContents()
+    sourceDir.copyRecursively(targetDir, overwrite = true)
+    // in each file replace the package name
+    for (file in targetDir.walkTopDown()) {
+      if (file.isFile && file.extension == "kt") {
+        val content = file.readText()
+        val newContent =
+            """
+        |/*
+        | * Generated file - DO NOT EDIT MANUALLY
```

Comment:
👍🏻

Reply:
We can btw add a similar comment to the top of the file in the code generator. My thinking was that we can just do `cp -r ...` in a bash script and ideally avoid custom gradle tasks like this one